### DISCUSSION TITLE
Asus Z390-F Gaming cannot be flashed with .CAP created by UEFITool and MMTool is nowhere to be found

### DISCUSSION THREAD

**Discussion by ACX-Com** [xCuri0/ReBarUEFI#246] [General]
**System**
 - Motherboard: Asus Z390-F Gaming
 - BIOS Version: 2102
 - GPU: Palit RTX 2080 Super Jetstream (ONLY RTX 2080, NOT RTX 2080 SUPER!!!)
 - [ x] CSM is turned off. YES
 - [ x] 4G decoding is enabled. YES
 - [ x] UEFIPatch is applied Tried both, but no Flashing is possible
 - [ x] I read. I found nothing about my config

**Description**

Additional info:
The board has the ReBar option in its own UEFI.
After I activated this, it runs on "Auto".
However, under Windows it is always deactivated and at 256MB.
When I try to use UEFITool AND (in further versions) then also UEFIPatch...
UEFIPatch reports "No patches can be applied to input file", so it makes no difference if I use it or not!?
I don't think I have any pad file problems, although I don't really understand where exactly I should look and what the difference is between the two example screenshots, except for the number and position of the pad file entries.
Problem:
I tried flashing with EZFlash (the one in the UEFI) and EZ Update (start in Windows).
By the way - the official UEFI 2102 works without any problems.
With the modified .CAB file it just works. When flashing the original .CAB file (start with EZ Update from Windows), Windows starts in EZFash (UEFI) and the flashing works. A direct start in EZFlash (UEFI) also works with the original .CAB file.
But when I use the modified .CAP file:
When starting from Windows, the system does not restart in EZ Flash, but the computer turns off (does not restart) and when I start it directly from UEFI EZFlash, I get the message that the file is not a real BIOS.

Can anyone help me get ReBar running on Z390-F Gaming with RTX 2080?

BR Achim

**Comment by mvildosolae** [comment]
Usually, built-in UEFI flashers like EZFlash or software from the OEM includes checksum verifications and other protections to prevent flashing non-authorised or modified firmware. Check [here](https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627) for alternatives that do not require an external SPI programmer.
